include/vcpkg/commands.build.h

```diff
@@ -71,40 +73,10 @@ namespace vcpkg
         CleanPackages clean_packages;
```

Comment:
@ras0219-msft suggested renaming the `editable` and `use_head_version` fields to add "`_if_user_requested`" here to make them clearer.